src/app.html

```diff
@@ -5,6 +5,18 @@
 		<title>Bounty Manager</title>
 		<link rel="icon" href="%sveltekit.assets%/favicon.svg" />
 		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta name="title" content="Polkadot Bounty Manager" />
```

Comment:
```suggestion
		<title>Polkadot Bounty Manager</title>
```
I believe this should be in the title tag above

Reply:
I only saw the comments now, for some reason I didn't get a notification. 
I have fixed and updated all of them now